content/kapp-controller/_index.html

```diff
@@ -55,108 +64,120 @@ <h2>Basic Usage</h2>
   serviceAccountName: default-ns-sa
   fetch:
   - git:
-      url: https://github.com/vmware-tanzu/carvel-simple-app-on-kubernetes
+      url: https://github.com/k14s/k8s-simple-app-example
       ref: origin/develop
-      subPath: config-step-1-minimal
+      subPath: config-step-2-template
   template:
   - ytt: {}
   deploy:
   - kapp: {}
-{{< / highlight >}}
-        <p>Let's apply configuration to your cluster and see it get reconciled.</p>
-        {{< highlight bash "" >}}
-$ kubect apply -f simple-app.yml
+{{< / highlight>}}
+    <p>Apply the App CR to your cluster and verify the deployment was successful:</p>
+{{< highlight bash "">}}
+$ kubectl get app simple-app
 
-$ kubectl get app
 NAME         DESCRIPTION           SINCE-DEPLOY   AGE
-simple-app   Reconcile succeeded   6s             28s
-{{< / highlight >}}
-        <p>Also take a look at more detailed information about deployed app via <code>kubectl get app simple-app -oyaml</code>.</p>
-        {{< highlight yaml "" >}}
-apiVersion: kappctrl.k14s.io/v1alpha1
-kind: App
+simple-app   Reconcile succeeded   9s             10s
+{{< / highlight>}}
+    <p>Discover more in the <a href="https://carvel.dev/kapp-controller/docs/latest/walkthrough/">App walkthrough</a> documentation.</p>    
+    <h3>Package authoring</h3>
+    <p>Want to start providing versioned software for Kubernetes for users to consume? Want to provide a simple interface for 
```

Comment:
This is a nit, but the "for Kubernetes for users" reads a bit weird to me. What if we did "versioned Kubernetes software for users" instead?